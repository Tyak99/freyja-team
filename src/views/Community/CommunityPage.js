import React from 'react';
import Community from '../../components/Community/Community';

export default () => <Community />;
