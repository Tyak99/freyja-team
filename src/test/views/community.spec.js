import React from 'react';
import { shallow } from 'enzyme';
import Community from '../../views/Community/CommunityPage';

describe('test community page', () => {
  const comm = shallow(<Community />);

  it('should test community page', () => {
    expect(comm).toMatchSnapshot();
  });
});
