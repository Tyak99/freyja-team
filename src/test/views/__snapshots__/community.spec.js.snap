// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test community page should test community page 1`] = `ShallowWrapper {}`;
